src/walletConnect/saga.ts

```diff
@@ -24,23 +22,21 @@ export function* isWalletConnectEnabled(uri: string) {
   return versionEnabled[version] ?? false
 }
 
-export function* initialiseWalletConnect(uri: string, origin: WalletConnectPairingOrigin) {
+export function* shouldInitialiseWalletConnect(uri: string, origin: WalletConnectPairingOrigin) {
```

Comment:
I feel the existing name is more appropriate.
It can fail and that's ok.
the `should` prefix seems to indicate it returns a boolean but doesn't do anything.

Reply:
I'll move this saga back down to the v2 file to have removing v1 and moving v2 to the root in different PRs. This should resolve the name conflict also.